include/af/graphics.h

```diff
@@ -1088,6 +1105,33 @@ AFAPI af_err af_set_axes_titles(const af_window wind,
                                 const af_cell* const props);
 #endif
 
+#if AF_API_VERSION >= 37
+/**
+   C Interface wrapper for setting axes labels formats for 2d charts
+
+   Passing correct value to \p zformat dictates the right behavior when it comes
```

Comment:
I think you should have a better description and then add that note at the bottom with the `\note` tag.